templates/includes/sidebar.html

```diff
@@ -0,0 +1,6 @@
+<ul>
+    <li><a href="{% url 'index' %}">Home page</a></li>

```

Comment:
Leave an empty href here for this step
```suggestion
    <li><a href="#">Home page</a></li>

```